<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlaylistSettingsPage</class>
 <widget class="QWidget" name="PlaylistSettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>516</width>
    <height>851</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Playlist</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="checkbox_warncloseplaylist">
     <property name="text">
      <string>Warn me when closing a playlist tab</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkbox_greyoutdeleted">
     <property name="text">
      <string>Grey out non existent songs in my playlists</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_1">
     <property name="title">
      <string>Using the menu to add a song will...</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_18">
      <item>
       <widget class="QComboBox" name="combobox_menuplaymode">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <item>
         <property name="text">
          <string>Never start playing</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Play if there is nothing already playing</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Always start playing</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupbox_previous">
     <property name="title">
      <string>Pressing &quot;Previous&quot; in player will...</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_19">
      <item>
       <widget class="QComboBox" name="combobox_previousmode">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <item>
         <property name="text">
          <string>Jump to previous song right away</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Restart song, then jump to previous if pressed again</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupbox_doubleclickaddmode">
     <property name="title">
      <string>Double clicking a song will...</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QComboBox" name="combobox_doubleclickaddmode">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <item>
         <property name="text">
          <string>Append to the playlist</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Replace the playlist</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Open in new playlist</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Add to the queue</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="combobox_doubleclickplaymode">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <item>
         <property name="text">
          <string>Never start playing</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Play if there is nothing already playing</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Always start playing</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupbox_doubleclickplaylist">
     <property name="title">
      <string>Double clicking a song in the playlist will...</string>
     </property>
     <layout class="QVBoxLayout" name="layout_doubleclick_playlist">
      <item>
       <widget class="QComboBox" name="combobox_doubleclickplaylistaddmode">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <item>
         <property name="text">
          <string>Change the currently playing song</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Add to the queue</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>When saving a playlist, file paths should be</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="radiobutton_automaticpath">
        <property name="text">
         <string>Auto&amp;matic</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radiobutton_absolutepath">
        <property name="text">
         <string>Absolute</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radiobutton_relativepath">
        <property name="text">
         <string>Relati&amp;ve</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radiobutton_askpath">
        <property name="text">
         <string>As&amp;k when saving</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_1">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupbox_editmetadatainline">
     <property name="title">
      <string>Metadata</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_editmetadatainline">
      <item>
       <widget class="QCheckBox" name="checkbox_editmetadatainline">
        <property name="toolTip">
         <string>If activated, clicking a selected song in the playlist view will let you edit the tag value directly</string>
        </property>
        <property name="text">
         <string>Enable song metadata inline edition with click</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkbox_writemetadata">
        <property name="text">
         <string>Write metadata when saving playlists</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
